packages/gestalt/src/headingStyles.js

```diff
@@ -0,0 +1,13 @@
+// @flow strict
```

Comment:
Created this to make future updates easier, but open to other format suggestions if this isn't ideal

Reply:
Can we compose this in a CSS file instead? Possibly in Heading.css?

Reply:
Always forget about compose, great call! Updated!